src/svelte/components/ListInput.svelte

```diff
@@ -217,28 +217,63 @@
             {min}
             {minlength}
             {step}
-            {multiple}
+            {...{ multiple }}
```

Comment:
Does this mean the value of `multiple` on the `select` element won't change if the parent changes the `multiple` value it's passing in? I.e., the parent can set it, but then can't change it?

Reply:
No, the component will update when the parent changes the value. This line just bypasses Svelte's static check.